Doc/library/unittest.rst

```diff
@@ -345,24 +346,6 @@ the `load_tests protocol`_.
    directory too (e.g.
    ``python -m unittest discover -s root/namespace -t root``).
 
-.. versionchanged:: 3.11
-   :mod:`unittest` dropped the :term:`namespace packages <namespace package>`
-   support in Python 3.11. It has been broken since Python 3.7. Start directory and
-   subdirectories containing tests must be regular package that have
-   ``__init__.py`` file.
-
-   Directories containing start directory still can be a namespace package.
-   In this case, you need to specify start directory as dotted package name,
-   and target directory explicitly. For example::
-
-      # proj/  <-- current directory
-      #   namespace/
-      #     mypkg/
-      #       __init__.py
-      #       test_mypkg.py
-
-      python -m unittest discover -s namespace.mypkg -t .
-
```

Comment:
do not remove this.
this note states that explicit specified namespace package is still supported.

Reply:
Thanks for the review. I am happy to keep whatever clarifying docs we need.

I left this in the revert because it suggests that `-t .` is required. But with this patch (which does more than just a straight revert, see subsequent commits), we have without `-t .`:

```py
% ../cpython/python.exe -m unittest discover -s namespace.mypkg
.
----------------------------------------------------------------------
Ran 1 test in 0.000s

OK
```